packages/atomic/src/components/atomic-result-template/atomic-result-template.tsx

```diff
@@ -6,10 +6,12 @@ import {
 } from '@coveo/headless';
 import {MapProp} from '../../utils/props-utils';
 import {containsSection} from '../../utils/result-section-utils';
+import {ResultDisplayImageSize} from '../atomic-result/atomic-result-display-options';
 
 export interface TemplateContent {
   innerHTML: string;
   usesSections: boolean;
+  image?: ResultDisplayImageSize;
```

Comment:
Could `imageSize` be a clearer name for this property? I feel `image` suggests a url?